jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibPlugin.java

```diff
@@ -193,9 +193,24 @@ public void apply(Project project) {
             jibDependencies.add(jarTask);
 
             if (projectAfterEvaluation.getPlugins().hasPlugin("org.springframework.boot")) {
-              Jar jar = (Jar) jarTask.get();
-              jar.setEnabled(true);
-              jar.getArchiveClassifier().set("original");
+              Task bootJarTask = projectAfterEvaluation.getTasks().getAt("bootJar");
```

Comment:
I think `getAt` is a method inherited from some parents class. I usually see the use of `.getByName("bootJar")` usually instead, unless there is some reason to use `getAt` instead?